client/lib/domains/registration/availability-messages.js

```diff
@@ -22,12 +24,15 @@ import {
 	domainTransferIn,
 } from 'my-sites/domains/paths';
 
-function getAvailabilityNotice( domain, error, site ) {
+function getAvailabilityNotice( domain, error, errorData ) {
 	let message,
 		severity = 'error';
 
 	const tld = getTld( domain );
 
+	const site = get( errorData, 'site', null );
```

Comment:
I think those two lines could be written like:
```const { site, maintenanceEndTime } = errorData;```

Reply:
Good point. 👍